src/Tribe/Main.php

```diff
@@ -4334,8 +4334,10 @@ public function addHelpAdminMenuItem() {
 		 * When the edit-tags.php screen loads, setup filters
 		 * to fix the tagcloud links
 		 *
+		 * @deprecated 4.1.2
 		 */
 		public function prepare_to_fix_tagcloud_links() {
+			_deprecated_function( __METHOD__, '4.1.2' );
```

Comment:
`__METHOD__` includes the class already, so this one would be: `Tribe__Events__Main::prepare_to_fix_tagcloud_links`